vagrantfiles/shell/install-nodejs.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+export DEBIAN_FRONTEND=noninteractive
+
+# see https://github.com/joyent/node/wiki/Installing-Node.js-via-package-manager#debian-and-ubuntu-based-linux-distributions
+apt-get install curl
+curl --silent --location https://deb.nodesource.com/setup_0.12 | bash -
```

Comment:
Using https://github.com/creationix/nvm is a better solution (like rvm but for nodejs)